src/components/TraceTree/d3-trace.js

```diff
@@ -222,7 +222,12 @@ export default class TraceMap {
       .attr('text-anchor', 'start')
       .attr('fill', d => this.type[d.data.layer])
       .attr('stroke', d => this.type[d.data.layer])
-      .text(d => d.data.layer);
+      .text(d => {
+        if(d.data.type == 'Local' && d.data.layer == 'Unknown'){
```

Comment:
Is that better?

Reply:
I think so.

Reply:
I think only checking the ''Unkown" is enough, using ternary statement to do this well.

Reply:
We can't just check `unknown`, because some RPC plugin may forget to set layer.